spec/features/items/index.html.erb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
```

Comment:
The extension on this file is wrong, should be `spec/features/items/index_spec.rb` :)